setup3.py

```diff
@@ -62,11 +62,12 @@ def main():
             'build_scripts': PickyBuildScripts,
         },
         packages=find_packages(),
-        conditionalExtensions=getExtensions(),
         install_requires=["zope.interface >= 4.0.2"],
-        zip_safe=False,
-        include_package_data=True,
+        conditionalExtensions=getExtensions(),
         scripts=getScripts(),
+        include_package_data=True,
+        zip_safe=False,
+        extras_require=_EXTRAS_REQUIRE,
```

Comment:
This is really minor, but this diff adds unnecessary churn here. Let's avoid re-ordering things that were already there (`conditionalExtensions`, `zip_safe`, and `include_package_data`), given that all we're doing is add `extras_require`.